### PR TITLE
Address deep review issues

### LINKED ISSUE #129: Reviewer 1 comment 1
> One minor suggestion/comment - it wasn't clear why the Manubot description came AFTER the Deep Review suggestion. Is it not true that DR was a first instance use of Manubot? If so, why not describe the tool, then describe DR as the pilot? Generally it read as though those two sections were distinct, while I think the authors intended that they be part of a bigger story.

### LINKED ISSUE #143: Reviewer 2 comment 14
> `We initiated the Deep Review` → when? Would be nice to give some date at this point.

### LINKED ISSUE #144: Reviewer 2 comment 15
> `In total, the Deep Review attracted...` → in less than a few weeks? months? years?

### PR DESCRIPTION
**Pull request by agitter**
Closes #129
Closes #143
Closes #144

My prior edit 9698331ebb8335dc3f6f73d196b7c9abb38111dd is also relevant here.  I tried to make it clear that the contribution workflow is about Manubot in general, Deep Review is only an example.

I'd still like to add a closing paragraph to the introduction about Manubot's features to further help with #129.

@vsmalladi I recommend adding a short paragraph about using Manubot on private repositories after the sentence `These pull request and contribution tracking examples both come from Deep Review, the largest Manubot project to date, but illustrate the general principles of transparency and collaboration that are shared by all open Manubot manuscripts.` when you work on #138.  You can transition with something like `Manubot can also be used on private manuscripts...`.

### CODE REVIEW

**Review by dhimmel — APPROVED** [review]

File: content/response-to-reviewers.md [hunk 1]
```diff
@@ -110,8 +115,16 @@ See [GH142](https://github.com/greenelab/meta-review/issues/142) and [GH170](htt
 > `We initiated the Deep Review` → when?
 Would be nice to give some date at this point.
 
+Deep Review as started in August 2016.
```
Comment (dhimmel) [inline content/response-to-reviewers.md]: Is this meant to be a quote?

File: content/response-to-reviewers.md [hunk 2]
```diff
@@ -110,8 +115,16 @@ See [GH142](https://github.com/greenelab/meta-review/issues/142) and [GH170](htt
...
+We discussed this comment in [GH143](https://github.com/greenelab/meta-review/pull/143)
+and made the requested change [GH183](https://github.com/greenelab/meta-review/pull/183).
+
 > `In total, the Deep Review attracted...` → in less than a few weeks? months? years?
 
+This referred to the less than two year period from August 2016 when the Deep Review project started to March 2018 when the journal submission was accepted.
+We discussed this comment in [GH143](https://github.com/greenelab/meta-review/pull/143)
+and made the requested change [GH183](https://github.com/greenelab/meta-review/pull/183).
```
Comment (dhimmel) [inline content/response-to-reviewers.md]: missing `in` before GH183

File: content/response-to-reviewers.md (hunk 1 above)
Comment (agitter) [reply content/response-to-reviewers.md]: It wasn't meant to be.  I fixed a typo and rephrased.

**Comment by agitter** [comment]
@dhimmel do you think I should add an extra paragraph to the end of the intro to highlight Manubot's features?  I like that the current intro is concise and to the point, but it spends more time on Deep Review than Manubot.

_Edit: this may also help with #130 and could be a separate pull request for that issue_

**Comment by dhimmel** [comment]
> do you think I should add an extra paragraph to the end

Yes I think that could be helpful, but doesn't have to be part of this PR. That may help with reader issues like https://github.com/greenelab/meta-review/issues/129.

**Comment by agitter** [comment]
> Yes I think that could be helpful, but doesn't have to be part of this PR.

Okay, I'll address that in a separate PR.  Is this ready to merge?

**Review by dhimmel — APPROVED** [review]
Yes merge it.

**Pull request merged**